meshtastic/interfaces/ble/receive_compat_service.py

```
@@ -0,0 +1,506 @@
+        )._wait_for_read_trigger(
+            coordinator=coordinator,
+            wait_timeout=wait_timeout,
+            wait_for_event=lambda target_coordinator, event_name, timeout: BLEReceiveRecoveryService._coordinator_wait_for_event(  # noqa: E501
+                target_coordinator,
+                event_name,
+                timeout=timeout,
+            ),
+            check_and_clear_event=lambda target_coordinator, event_name: BLEReceiveRecoveryService._coordinator_check_and_clear_event(  # noqa: E501
+                target_coordinator,
+                event_name,
+            ),
```

Ruff (0.15.6)

[warning] Lambda may be unnecessary; consider inlining inner function

Inline function call

(PLW0108)